config/install/language.entity.bg.yml

```diff
@@ -1,6 +1,8 @@
 langcode: en
 status: true
-dependencies: {  }
+dependencies:
```

Comment:
Since third party settings are provided then need to add a dependency. Drupal added it automatically.